index.js

```diff
@@ -153,134 +52,81 @@ function getSha(imageName) {
-  }
-  return logs;
-}
-
-async function getContents(owner, repo, path) {
-  const { data: data } = await octokit.repos.getContents({
-    owner,
-    repo,
-    path,
-  });
-  return data;
-}
-
-async function createPR(
-  projects,
-  issueContent,
-  releaseConfig,
-  newReleaseContentBlob
-) {
-  const branchName = `release-${new Date().getTime()}`;
-  const manifestsSha = await getHeadSha("notification-manifests");
-  const logs = await buildLogs(projects);
-
-  const ref = await octokit.git.createRef({
-    owner: GH_CDS,
-    repo: "notification-manifests",
-    ref: `refs/heads/${branchName}`,
-    sha: manifestsSha,
-  });
-
-  const manifestUpdates = projects
-    .map((project) => {
-      return `${project.name}:${shortSha(project.headSha)}`;
-    })
-    .join(" and ");
-  const update = await octokit.repos.createOrUpdateFile({
-    owner: GH_CDS,
-    repo: "notification-manifests",
-    branch: branchName,
-    sha: releaseConfig.sha,
-    path: "env/production/kustomization.yaml",
-    message: `Updated manifests to ${manifestUpdates}`,
-    content: newReleaseContentBlob,
-  });
-
-  const pr = await octokit.pulls.create({
-    owner: GH_CDS,
-    repo: "notification-manifests",
-    title: `[AUTO-PR] Automatically generated new release ${new Date().toISOString()}`,
-    head: branchName,
-    base: "main",
-    body: issueContent.replace(
-      "> Give details ex. Security patching, content update, more API pods etc",
-      logs
-    ),
-    draft: true,
-  });
-  return Promise.all([ref, update, pr]);
-}
-
-async function hydrateWithSHAs(releaseConfig, projects) {
+async function hydrateWithSHAs() {
   return await Promise.all(
-    releaseConfig.images.map(async (image) => {
-      const matchingProject = projects.find((project) =>
-        image.newName.includes(project.ecrName)
+    PROJECTS.map(async (project) => {
+      project.headSha = await getHeadSha(project.repoName);
+      project.shortSha = shortSha(project.headSha)
+      project.headUrl = getLatestImageUrl(
+        PROJECTS,
+        project.ecrName,
+        project.headSha
       );
-      matchingProject.headSha = await getHeadSha(matchingProject.name);
-      matchingProject.headUrl = getLatestImageUrl(
-        matchingProject.ecrName,
-        matchingProject.headSha
+
+      const releaseContent = await getContents(
+        "notification-manifests",
+        project.manifestFile
       );
-      matchingProject.oldSha = getSha(image.newName);
-      matchingProject.oldUrl = image.newName;
-      image.newName = matchingProject.headUrl;
-      return matchingProject;
+
+      const originalFileContents = Base64.decode(releaseContent.content)
+      const re = new RegExp(`${project.ecrName}:\\S*`, "g");
+      project.oldUrl = originalFileContents.match(re)[0]
+      project.oldSha = getSha(project.oldUrl);
+      return project;
     })
   );
 }
 
+function UpdateContents(content, project) {
```

Comment:
Let's use Javascript standard of lowerCamelCase format for Javascript function name please.